src/Adapter/Cart/CartPresenter.php

```diff
@@ -295,12 +295,13 @@ public function present($cart, $shouldSeparateGifts = false)
         $total_excluding_tax = $cart->getOrderTotal(false);
         $total_including_tax = $cart->getOrderTotal(true);
         $total_discount = $cart->getDiscountSubtotalWithoutGifts();
+        $totalCardAmount = $cart->getOrderTotal($this->includeTaxes(), Cart::ONLY_PRODUCTS);
```

Comment:
Even if it won't break anything, I guess you meant `$totalCartAmount` instead of `$totalCardAmount`